.github/workflows/docscheck.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       - uses: actions/checkout@v3
       - uses: rojopolis/spellcheck-github-actions@v0
         name: Spellcheck
+      - uses: rdohms/textlint-action@latest
+        with:
+          rulesets: 'textlint-filter-rule-comments textlint-rule-no-todo textlint-rule-stop-words textlint-rule-terminology'
+          path: '{README.md,version-history.md,docs/**}'
```

Comment:
IMO version-history.md is not something that needs to be checked at all. Just meant extra work for you in this. It's *not* included in docs because it corrupts the search so much.

Reply:
I don’t mind since it’s both public-facing and important, and I figure it should be held to the same standard as writing in the documentation. (Same with the developer docs this also fixes—though the spell checker would go wild there if we stopped ignoring that section.)

Are you saying you’d like me to remove that from the check? The fixing-up is already done so after this PR it’s only a matter of maintenance/conformance.

Reply:
Thanks for checking the box. The other option would have been to remove the line, which would have been completely and historically correct.

Reply:
Whoops! I was going for correctness—removed the line instead. The fact that we’re discussing style and accuracy is a reason to include this file, IMO!